app/templates/main.html

```diff
@@ -1,7 +1,18 @@
 <section>
 	<div>
+		<form>
+			<label>Add a Room!<br>
+			<input type="text" name="add_room" ng-model="roomName" placeholder="Room Name"></label>
+			<input type="button" value="Add Room!" ng-click="main.addRoom(roomName)">
+		</form>
 		<ul ng-repeat="room in rooms">
 			<li>{{room.$value}}</li>
+			<!-- An idea I want to do is have the name of the new room bind instantly to a new dom element, only to destroy itself when user submits form - so it appears to the user their room is being named in real time, even though it's only instantialized upon form submission. Far fetched? -->
```

Comment:
What I'd think about is complexity vs benefit to user to implement it. If the benefit is greater, do it. It's your call.